cron/internal/worker/main.go

```diff
@@ -128,11 +129,21 @@ func processRequest(ctx context.Context,
 		if err := format.AsJSON2(&result, true /*showDetails*/, log.InfoLevel, checkDocs, &buffer2); err != nil {
 			return fmt.Errorf("error during result.AsJSON2: %w", err)
 		}
+		exportPath := fmt.Sprintf("%s/result.json", repo.URI())
+		exportCommitSHAPath := fmt.Sprintf("%s/%s/result.json", repo.URI(), result.Repo.CommitSHA)
 
 		// Raw result.
 		if err := format.AsRawJSON(&result, &rawBuffer); err != nil {
 			return fmt.Errorf("error during result.AsRawJSON: %w", err)
 		}
+
+		if err := data.WriteToBlobStore(ctx, exportBucketURL, exportPath, buffer2.Bytes()); err != nil {
+			return fmt.Errorf("error during WriteToBlobStore2: %w", err)
+		}
+		// Export result based on commitSHA.
+		if err := data.WriteToBlobStore(ctx, exportBucketURL, exportCommitSHAPath, buffer2.Bytes()); err != nil {
+			return fmt.Errorf("error during WriteToBlobStore2: %w", err)
+		}
```

Comment:
IIUC, `buffer2` will contain appended data from all repos. You'll need a separate var to get the data belonging only to this repo.

Reply:
I have fixed it. Thanks